steamworks_windows.go

```diff
@@ -3,6 +3,9 @@
 
 package steamworks
 
+// #include <stdlib.h>
+import "C"
```

Comment:
Don't use Cgo. Instead, we can add this code:

```go
ckey := append([]byte(key), 0)
defer runtime.KeepAlive(ckey)
```

See `ClearAchievement` for example.